src/Microsoft.Extensions.Configuration.AzureAppConfiguration/AzureAppConfigurationProvider.cs

```diff
@@ -970,7 +970,8 @@ private void SetRequestTracingOptions()
                 IsKeyVaultConfigured = _options.IsKeyVaultConfigured,
                 IsKeyVaultRefreshConfigured = _options.IsKeyVaultRefreshConfigured,
                 ReplicaCount = _options.Endpoints?.Count() - 1 ?? _options.ConnectionStrings?.Count() - 1 ?? 0,
-                FeatureFlagTracing = _options.FeatureFlagTracing
+                FeatureFlagTracing = _options.FeatureFlagTracing,
+                IsLoadBalancingEnabled = _options.LoadBalancingEnabled
```

Comment:
We missed adding this to the telemetry. @zhiyuanliang-ms could you create another PR to add telemetry when load balancing is enabled?